main.nf

```diff
@@ -360,7 +367,7 @@ if(params.aligner == 'bismark'){
         input:
         set val(name), file(reads) from trimmed_reads
         file index from bismark_index.collect()
-        file wherearemyfiles
+        file wherearemyfiles from wherearemyfiles_2
```

Comment:
Same here, please use explicit names for channels :-)
It gets super confusing at some point otherwise....